### PR TITLE
docs: changelog v5.4.1

### PR DESCRIPTION
**Pull request by vagusX**
Changelogs for v5.4.1

---------------------------------

### <samp>🤖 Generated by Copilot at 828558f</samp>

This pull request updates ant-design to version 5.4.1 and adds the corresponding release notes in English and Chinese. It fixes some bugs, improves some styles, and updates some locales in the library.

### <samp>🤖 Generated by Copilot at 828558f</samp>

*  Update the version number of ant-design to 5.4.1 in `package.json` ([link](https://github.com/ant-design/ant-design/pull/41734/files?diff=unified&w=0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L3-R3))
*  Add the release notes for version 5.4.1 in English and Chinese in `CHANGELOG.en-US.md` and `CHANGELOG.zh-CN.md` respectively ([link](https://github.com/ant-design/ant-design/pull/41734/files?diff=unified&w=0#diff-e961ad288c43a2feb2d56b696e10c0b670d652dbb17c76c9aa0452d45157977cR18-R38), [link](https://github.com/ant-design/ant-design/pull/41734/files?diff=unified&w=0#diff-1224d5ca5233fdeba67ba2e2dc6fa3e615905253f35dc132c4b4b86f51f9eab4R18-R38))

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
+- 🐞 修复 Space 影响父元素字体大小和样式的问题。[#40326](https://github.com/ant-design/ant-design/pull/40326)
+- 🐞 修复 Pagination 简洁模式中的上一页下一页按钮 hover 样式丢失的问题。[#41685](https://github.com/ant-design/ant-design/pull/41685)
+- 🐞 修复 Tree `switcherIcon` 无法隐藏。[#41708](https://github.com/ant-design/ant-design/pull/41708) [@acyza](https://github.com/acyza)
+- 🐞 修复 `List.Item.Meta` 的 avatar 和 title 不对齐。[#41688](https://github.com/ant-design/ant-design/pull/41688) [@Yuiai01](https://github.com/Yuiai01)
+- 🐞 修复 Row 的 `justify` 设置为 `space-evenly` 无效。[#41679](https://github.com/ant-design/ant-design/pull/41679) [@Yuiai01](https://github.com/Yuiai01)
+- 🐞 修复 Button props 的类型问题。[#41650](https://github.com/ant-design/ant-design/pull/41650)
+- 🐞 修复表格列宽不够的问题 [issue 40814](https://github.com/ant-design/ant-design/issues/40814)。[#41626](https://github.com/ant-design/ant-design/pull/41626)
+- 🐞 修复 Mentions 弹层样式。[#41660](https://github.com/ant-design/ant-design/pull/41660)
+- 🐞 修复弹出 Modal 后，若鼠标未经过 header，选择 body 内容时会移动 Modal 的问题。[#41630](https://github.com/ant-design/ant-design/pull/41630) [@chutao-zhang](https://github.com/chutao-zhang)
```
Comment (li-jia-nan) [inline CHANGELOG.zh-CN.md:32]: 这个是demo，应该不用写进去吧

Comment (afc163) [reply CHANGELOG.zh-CN.md:32]: +1

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,26 @@ timeline: true
...
+- 🐞 修复 Select 在搜索时宽度变为 `0px` 的问题。[#41722](https://github.com/ant-design/ant-design/pull/41722)
+- 🐞 修复 Empty 空数据组件在宽度不够的容器内样式错位的问题。[#41727](https://github.com/ant-design/ant-design/pull/41727)
+- 🐞 改进 Form.Item `noStyle` 验证消息显隐逻辑。[#41698](https://github.com/ant-design/ant-design/pull/41698) [@Wxh16144](https://github.com/Wxh16144)
+- 🐞 修正 Form.Item 不应支持设置 `requiredMark` 的问题。[#41725](https://github.com/ant-design/ant-design/pull/41725) [@Wxh16144](https://github.com/Wxh16144)
+- 🐞 修复 Space 影响父元素字体大小和样式的问题。[#40326](https://github.com/ant-design/ant-design/pull/40326)
+- 🐞 修复 Pagination 简洁模式中的上一页下一页按钮 hover 样式丢失的问题。[#41685](https://github.com/ant-design/ant-design/pull/41685)
+- 🐞 修复 Tree `switcherIcon` 无法隐藏。[#41708](https://github.com/ant-design/ant-design/pull/41708) [@acyza](https://github.com/acyza)
+- 🐞 修复 `List.Item.Meta` 的 avatar 和 title 不对齐。[#41688](https://github.com/ant-design/ant-design/pull/41688) [@Yuiai01](https://github.com/Yuiai01)
+- 🐞 修复 Row 的 `justify` 设置为 `space-evenly` 无效。[#41679](https://github.com/ant-design/ant-design/pull/41679) [@Yuiai01](https://github.com/Yuiai01)
+- 🐞 修复 Button props 的类型问题。[#41650](https://github.com/ant-design/ant-design/pull/41650)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:29]: 修复 Button 配置 `data-*` 属性的 TS 定义。

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,26 @@ timeline: true
...
+- 🐞 修复表格列宽不够的问题 [issue 40814](https://github.com/ant-design/ant-design/issues/40814)。[#41626](https://github.com/ant-design/ant-design/pull/41626)
+- 🐞 修复 Mentions 弹层样式。[#41660](https://github.com/ant-design/ant-design/pull/41660)
+- 🐞 改进 Form.Item 关于 `require` 的判断逻辑。[#41623](https://github.com/ant-design/ant-design/pull/41623) [@Wxh16144](https://github.com/Wxh16144)
+- 💄 去除 Badge.Ribbon 里固定的高度。[#41661](https://github.com/ant-design/ant-design/pull/41661) [@MuxinFeng](https://github.com/MuxinFeng)
+- 💄 优化类 Select 组件弹窗逻辑，现在总是会尝试优先在可视区域展示以减少用户额外滚动成本。[#41619](https://github.com/ant-design/ant-design/pull/41619)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:34]: 这个应该放第一个。

**Comment by li-jia-nan** [comment]
好像没写日期？

File: CHANGELOG.zh-CN.md (hunk 3 above)
Comment (zombieJ) [reply CHANGELOG.zh-CN.md:34]: 就是位置挪到最上面

**Review by zombieJ — APPROVED** [review]

File: CHANGELOG.zh-CN.md (hunk 3 above)
Comment (afc163) [reply CHANGELOG.zh-CN.md:34]: 这个是本版本最重要的改动。

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+- 🐞 修复 Empty 空数据组件在宽度不够的容器内样式错位的问题。[#41727](https://github.com/ant-design/ant-design/pull/41727)
+- 🐞 改进 Form.Item `noStyle` 验证消息显隐逻辑。[#41698](https://github.com/ant-design/ant-design/pull/41698) [@Wxh16144](https://github.com/Wxh16144)
+- 🐞 修正 Form.Item 不应支持设置 `requiredMark` 的问题。[#41725](https://github.com/ant-design/ant-design/pull/41725) [@Wxh16144](https://github.com/Wxh16144)
+- 🐞 修复 Space 影响父元素字体大小和样式的问题。[#40326](https://github.com/ant-design/ant-design/pull/40326)
+- 🐞 修复 Pagination 简洁模式中的上一页下一页按钮 hover 样式丢失的问题。[#41685](https://github.com/ant-design/ant-design/pull/41685)
+- 🐞 修复 Tree `switcherIcon` 无法隐藏。[#41708](https://github.com/ant-design/ant-design/pull/41708) [@acyza](https://github.com/acyza)
+- 🐞 修复 `List.Item.Meta` 的 avatar 和 title 不对齐。[#41688](https://github.com/ant-design/ant-design/pull/41688) [@Yuiai01](https://github.com/Yuiai01)
+- 🐞 修复 Row 的 `justify` 设置为 `space-evenly` 无效。[#41679](https://github.com/ant-design/ant-design/pull/41679) [@Yuiai01](https://github.com/Yuiai01)
+- 🐞 修复 Button props 类型定义支持 data-\* 属性。[#41650](https://github.com/ant-design/ant-design/pull/41650)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:33]: 修复 Button 定义不支持 `data-*` 属性的问题。

File: CHANGELOG.zh-CN.md [hunk 5]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+- 🐞 修复表格列宽不够的问题 [issue 40814](https://github.com/ant-design/ant-design/issues/40814)。[#41626](https://github.com/ant-design/ant-design/pull/41626)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:34]:
```suggestion
- 🐞 修复 Table `rowSelection.selections` 有值时选择列列宽不够的问题。[#41626](https://github.com/ant-design/ant-design/pull/41626)
```

**Pull request merged**